test/TEST-01-BASIC/test.sh

```diff
@@ -6,6 +6,7 @@ TEST_DESCRIPTION="Basic systemd setup"
 IMAGE_NAME="basic"
 RUN_IN_UNPRIVILEGED_CONTAINER=${RUN_IN_UNPRIVILEGED_CONTAINER:-yes}
 TEST_REQUIRE_INSTALL_TESTS=0
+TEST_MASK_SERVICES=0
```

Comment:
I think TEST-01-BASIC is supposed to test all services so `TEST_MASK_SERVICES` should probably be `1` here and `0` everywhere else.

Reply:
Looks like the name is confusing. It isn't about testing those services. It's about masking them :-)

Reply:
I think I'd rename it to `TEST_SUPPORTING_SERVICES_SHOULD_BE_MASKED` to make it clear what it does but I think it can be done later.

Reply:
Hehe, yeah, I'm bad at naming things, I'll change it to `TEST_SUPPORTING_SERVICES_SHOULD_BE_MASKED` :-)